book/docs/classes/OpusPool/Constructor/constructor.md

```diff
@@ -0,0 +1,24 @@
+### constructor
```

Comment:
I suggest letting this file on its own, not in a dir. As a user I would expect to just see the constructor when I click `constructor` [here](https://chorus-one.gitbook.io/opus-pool-sdk-1.0/~/revisions/9fsgnavxfb0s143LvTjt/api-reference/opuspool)